index.html

```diff
@@ -36,7 +36,217 @@ <h2 class="sd">
+                    <div class="button-cont">
+                        <input type="button" value="see project" class="potfolio-but">
+                    </div>
+
+                    </div>
+                </div>
+            <div class="container-card">
+            <div class="placeholder">
                 
+            </div>
+            <div class="card-lists">
+                <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+                <div class="items">
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>Css</li>
+                    <li>Java script</li>
+                    <li>Html</li>
+                </ul>
+            </div>
+            <div class="button-cont">
+                <input type="button" value="see project" class="potfolio-but">          
+              </div>
+            </div>
+        </div>
+    <div class="container-card">
+    <div class="placeholder">
+        
+    </div>
+    <div class="card-lists">
+        <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+        <div class="items">
+        <ul>
+            <li>Ruby on rails</li>
+            <li>Css</li>
+            <li>Java script</li>
+            <li>Html</li>
+        </ul>
+    </div>
+    <div class="button-cont">
+        <input type="button" value="see project" class="potfolio-but">
+    </div>
+    </div>
+</div>
+<div class="container-card">
+<div class="placeholder">
+    
+</div>
+<div class="card-lists">
+    <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+    <div class="items">
+    <ul>
+        <li>Ruby on rails</li>
+        <li>Css</li>
+        <li>Java script</li>
+        <li>Html</li>
+    </ul>
+</div>
+<div class="button-cont">
+    <input type="button" value="see project" class="potfolio-but">
+</div>
+</div>
+</div>
+<div class="container-card">
+<div class="placeholder">
+    
+</div>
+<div class="card-lists">
+    <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+    <div class="items">
+    <ul>
+        <li>Ruby on rails</li>
+        <li>Css</li>
+        <li>Java script</li>
+        <li>Html</li>
+    </ul>
+</div>
+<div class="button-cont">
+    <input type="button" value="see project" class="potfolio-but">
+</div>
+</div>
+</div>        
+<section class="about-me">
+  <div> 
+      <div class="pic3" > 
+           <a href="#"><img src="./images/pointer.png" alt=""> </a></div>
+     <h1>About Me</h1>
+    <p> I can help you build a product , 
+          feature or website Look through 
+        some of my work and experience! If 
+           you like what you see and have a 
+          project you need coded, 
+              don’t hestiate to contact me.
+       </p>
+           </div>
+         <div class="bot-container">
+      <div class="bot-img">
+         <a href="#"><img src="./images/Group.png" alt=""></a>
+        </div>
+        <div class="botton-con">
+            <input type="button" value="see project" class="potfolio-but">
+        </div>
```

Comment:
- [ ] kindly refer to what I said in the previous comment about the `see project` button